galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/ArtifactPropertiesArtifact.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright contributors to the Galasa project
+ */
+package dev.galasa.framework.api.ras.internal;
```

Comment:
Might be worth adding a bit more structure to this package...
We have base/common stuff, but can group things by ras/run/get ras/run/download ...etc ?